.editorconfig

```diff
@@ -6,8 +6,8 @@ charset = utf-8
 trim_trailing_whitespace = true
 insert_final_newline = true
 
-[*.cs]
-indent_size = 4
+[*.yaml]
+indent_size = 2
 indent_style = space
```

Comment:
Would it make sense to use `indent_style = space` as a default setting `[*]`?

Reply:
@ckr123 The first tooling we used would apply the settings to everything including binary files, so I would suggest only configuring what we want to configure.